README.md

```diff
@@ -52,29 +52,33 @@
    e. To manage any funds of the organization.
    f. To maintain an active log of the project's income and expenditures.
    g. To serve as the lead organizer for all fundraisers.
+11. The duties of the General members shall be:
+    a. To attend meetings and be informed about the operations of the Organization.
+    b. To support the committee in its operations and decision-making.
 
 ## ARTICLE V: FUNDING
 
 1. The organization shall be funded through the following means: fiscal sponsorship, fundraisers, contributions from any organization or individual that supports the Project's mission, fundraisers, and any approved financial assistance granted by the Project's steering committee.
 2. All money transactions must be approved by the Treasurer.
 3. The Steering Committee shall maintain a record of all financial transactions and accounts. These records shall be made available at any time by the request of any active member of the Project and the general public.
+4. All money transactions over $100 must be approved by a majority vote at a regular meeting.
+5. All Steering Committee members shall be added to the HCB organization.
 
 ## ARTICLE VI: MEETINGS
 
-1. Regular meetings of the organization shall be held monthly. The day of the week and time will be determined via slack, and updated as needed on a regular basis.
-   a. Meeting times may be changed by a two-thirds (⅔) vote of membership.
-2. Special meetings may be called by the Steering Committee. Votes and elections may not occur at such meetings.
+1. Regular meetings of the organization shall be held on the third Sunday of each calendar month.
```

Comment:
We should add some flexibility to this! and add something like "this can change if needed by the steering commitee." like what happened when it was delayed for this meeting